src/components/jobDetailsComponent/userButtons.tsx

```diff
@@ -83,16 +83,14 @@ const UserButtons: React.FC = () => {
   };
   return (
     <div className="jobDetailsButtons">
-      {job.reservedUser === user.id && (
-        <Button variant="outlined" onClick={handleReserve}>
-          {job.reserved ? (
-            <CloseIcon className="buttonIcon" />
-          ) : (
-            <DoneIcon className="buttonIcon" />
-          )}
-          {job.reserved ? 'Cancel' : 'Reserve'}
-        </Button>
-      )}
+      <Button variant="outlined" onClick={handleReserve}>
```

Comment:
why did you change this? cancel and reserve should only show if the person seeing the page is the one who reserved the job.

Reply:
Oh there was a bug when we were first going through it that the button was never showing for people to reserve the job